ci/travis/build-docker-images.py

```diff
@@ -0,0 +1,260 @@
+import datetime
+import functools
+import glob
+import os
+import re
+import runpy
+import shutil
+import sys
+from contextlib import redirect_stdout
+from io import StringIO
+from typing import List
+
+import docker
+
+print = functools.partial(print, file=sys.stderr, flush=True)
+DOCKER_USERNAME = "raytravisbot"
+DOCKER_CLIENT = None
+PYTHON_WHL_VERSION = "cp37m"
+
+
+def _merge_build():
+    return os.environ.get("TRAVIS_PULL_REQUEST") == "false"
+
+
+def _release_build():
+    branch = os.environ.get("TRAVIS_BRANCH")
+    if not branch:
+        print("Branch not found!")
+        print(os.environ)
+        print("Environment is above ^^")
+        return False
+    return branch != "master" and "releases" in branch
```

Comment:
Hmm, we could maybe do that--but I would prefer with keeping with the CI environment.

Reply:
hmmm...not sure I understand. I was referring to the `!= "master"` because if we decide to rename the main branch from `master` to `main`, we'll need to remember to update this line. Not really a minor issue, though: definitely something to save for later (and changing branches will probably be a 1 time thing so as long as we remember we shouldn't have any problems).

resolving this comment because this feature seems a little unnecessary tbh